auv_control/auv_control/launch/start.launch

```diff
@@ -36,7 +36,6 @@
     <node pkg="auv_control" type="depth_controller_node.py" name="depth_controller_node"
       output="screen">
       <param name="update_rate" value="$(arg rate)" />
```

Comment:
I think you'll need the topic remappings. Even if the topics are 1:1 matched, we still do remap, example: from odometry to odmetry

Keep in mind the control enable handler subscribes to "control" topic too, so make sure you include that as well.